pkg/vet/cmd/vet.go

```diff
@@ -97,7 +96,8 @@ func vet(cmd *cobra.Command, args []string) error {
 		vetter.Vetter(serviceassociation.NewVetter(informerFactory)),
 		vetter.Vetter(danglingroutedestinationhost.NewVetter(informerFactory)),
 		vetter.Vetter(conflictingvirtualservicehost.NewVetter(informerFactory)),
-		vetter.Vetter(invalidserviceforjwtpolicy.NewVetter(informerFactory)),
+		// obsolete in Istio 1.5
+		// vetter.Vetter(invalidserviceforjwtpolicy.NewVetter(informerFactory)),
```

Comment:
Is there a reason for commenting this out instead of removing it outright?

Reply:
Only reason is because there is a comment about it being no longer relevant to Istio 1.5; guess it is redundant w/ PR description, so removing.   Thanks!